ci/jenkins/test-rancher.sh

```diff
@@ -0,0 +1,319 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Antrea Authors
```

Comment:
```suggestion
# Copyright 2023 Antrea Authors
```
Make sure you use 2023 for any new file.

Reply:
@XinShuYang @edwardbadboy I removed this from v1.11 milestone today, let me know if we need add it back if this will block any other task.